packages/gatsby-plugin-utils/README.md

```diff
@@ -6,10 +6,38 @@
 npm install gatsby-plugin-utils
 ```
 
-### Example
+### validateOptionsSchema
+
+#### example
 
 ```js
 import { validateOptionsSchema } from "gatsby-plugin-utils"
 
 await validateOptionsSchema(pluginName, pluginSchema, pluginOptions)
 ```
+
+### testPluginOptionsSchema
+
+Utility to validate and test plugin options schemas. An example of a plugin options schema implementation can be found in the [gatsby-node.js file of gatsby-plugin-google-analytics](../gatsby-plugin-google-analytics/gatsby-node.js).
```

Comment:
This won't always be seen in the context of the monorepo. e.g. it might be in npm. Better to use an absolute URL

Reply:
Great catch, thank you 👍